Makefile

```diff
@@ -310,7 +320,13 @@ bundle-clean: ## Clean generated bundle files
 
 .PHONY: bin-clean
 bin-clean: ## Clean downloaded binaries
+	@[ -d bin ] && chmod -R u+rwx bin || true
 	$(RM) -r bin
 
+.PHONY: catalog-clean
+catalog-clean: ## Clean catalog files
+	$(RM) -r catalog
```

Comment:
This needs to be removed once the catalog dir has been checked in to the repo.

Reply:
Both `catalog` initial creation and removal should be dropped from the rules in this case.

Reply:
removing catalog-init/catalog-clean from Makefile
Adding directions to generate catalog to DEVELOPMENT.md

Reply:
Dropped